valhalla/mjolnir/osmway.h

```diff
@@ -2645,7 +2629,7 @@ struct OSMWay {
   uint32_t tunnel_ : 1;
   uint32_t toll_ : 1;
   uint32_t bridge_ : 1;
-  uint32_t seasonal_ : 1;
+  uint32_t UNUSED_seasonal_ : 1; // can be reclaimed
```

Comment:
can you just put it back to the spare nomenclature to match the others?
```suggestion
  uint32_t spare1_ : 1;
```